cognee/modules/pipelines/operations/run_tasks.py

```
@@ -1,226 +1,24 @@
-
 from typing import Any
+from uuid import UUID, NAMESPACE_OID, uuid4, uuid5
+
 from cognee.modules.pipelines.operations import (
     log_pipeline_run_start,
     log_pipeline_run_complete,
     log_pipeline_run_error,
 )
-from cognee.modules.settings import get_current_settings
-from cognee.modules.users.methods import get_default_user
 from cognee.modules.users.models import User
```

Ruff (0.8.2)

`cognee.modules.users.models.User` imported but unused

Remove unused import: `cognee.modules.users.models.User`

(F401)